docs/conf.py

```diff
@@ -97,24 +97,19 @@ def get_tutorial_images():
     mslib.mswms.wms.server.generate_gallery(sphinx=True, generate_code=True, all_plots=True, levels="3,4,200,300",
                                             vtimes="2012-10-18T00:00:00,2012-10-19T00:00:00")
 
-# readthedocs has no past.builtins
-try:
-    from past.builtins import execfile
-except ImportError as ex:
-    logging.error("%s", ex)
-execfile('../mslib/version.py')
+from mslib.version import __version__
```

Comment:
there is not an mss installation on readthedocs

So it can't become imported

Reply:
I am not sure if we would exceed a limit on readthedocs with installing MSS. The env gets large, May has selected in the evironment.yml the needed modules for a gallery build. We could also use a copy of the _load_module of https://github.com/Open-MSS/MSS/blob/develop/conftest.py#L208

When it is merged and can't be builded the old version can't be updated and is kept.

Reply:
It is installed without any dependencies (since the python package declares none), so it shouldn't be any larger than the repository that is already cloned.

If you really don't want it installed in that environment, and `__version__` is the only thing we need from the mslib module, then I would move the value of `mslib.version.__version__` into a separate `VERSION` file and just read it from there.

What in https://github.com/Open-MSS/MSS/blob/develop/conftest.py#L208 is IMO too complex for such a simple thing, and I don't even know what its purpose is in conftest.py and suspect that it could, and should, be removed there as well.